README.md

```diff
@@ -104,39 +104,91 @@ dropdown boxes.
+
+In the new API, all positional arguments from the old API can be utilized as keyword arguments. If both positional and keyword arguments are supplied, positional arguments will be disregarded.
 
 #### Selecting Machines
 
-The easiest way to get a list of Machines is with:
+The most straightforward way to obtain a list of Machines is by utilizing either of the two APIs:
+
+#### Old API:
+
+```
+cli.get_machine_names(machine_type)
+```
+
+#### New API:
 
 ```
 cli.get_machine_names(source_type=machine_type)
 ```
 
-The *machine_type* variable would be a machine type name retrieved from a previous step.  The `source_type` argument is optional.  If you do not provide it, then
-this function will return all machine names in the system.
+The *machine_type* variable would be a machine type name retrieved from a previous step.  The `source_type` argument is optional.  If you do not provide it, then this function will return all machine names in the system.
+
+In the new API, all the positional arguments from the old API can be used as keyword arguments. If both positional arguments and keyword arguments are given, positional arguments will be neglected.
+
+#### Machine Types
+
+The most straightforward way to obtain a list of Machine Types and associated Metadata is by utilizing either of the two APIs:
+
+#### Old API:
+
+```
+cli.get_machine_types(machine_type)
+```
+
+#### New API:
+
+```
+cli.get_machine_types(source_type=machine_type)
+```
+
+The *machine_type* variable would be a machine type name retrieved from a previous step.  The `source_type` argument is optional.  If you do not provide it, then this function will return all machine types and associated metadata in the system.
+
+In the new API, all the positional arguments from the old API can be used as keyword arguments. If both positional arguments and keyword arguments are given, positional arguments will be neglected.
 
 #### Machine Schema
 
-Although not requred, it is sometimes helpful to get a list of all stats available for a given machine.  That is done with the `get_machine_schema` function.  
+Although not requred, it is sometimes helpful to get a list of all stats available for a given Machine.  That is done with the any one of the following SDK function calls:  
+
+##### Old API:
 
 ```
 cli.get_machine_schema(machine_name)
 ```
 
+##### New API:
+
+```
+cli.get_machine_schema(*args, **query)
+```
+
 This returns a Pandas data frame with each stats Sight Machine internal name, display name, and type.  It takes an optional `types` argument, which filters the returned list to only a specific set of data types.  For example, to get the display names of all numeric stats on a machine, use:
 
 ```
 cli.get_machine_schema(machine_name, types=['int', 'float'])['display'].tolist()
 ```
 
+In the new API, all the positional arguments from the old API can be used as keyword arguments. If both positional arguments and keyword arguments are given, positional arguments will be neglected.
```

Comment:
Example of output and which positional arguments

Reply:
Added examples.